src/marvin/bot/base.py

```diff
@@ -729,10 +730,13 @@ def _reformat_response(
     target_return_type: Any,
     error_message: str,
 ) -> str:
+    try:
+        return json.dumps(ast.literal_eval(llm_response))
+    except ValueError:
+        pass
     @marvin.ai_fn(
         plugins=[],
         response_format=JSONFormatter(on_error="ignore"),
-        llm_model="gpt-3.5-turbo",
```

Comment:
Noticed this harded `gpt`, which caused my VertexAI calls break whenever they went through retries.